### PR TITLE
updated inlineRequire entry in README.md to reflect file-loader loading ES Modules by default

### PR DESCRIPTION
**Pull request by juliankrieger**
Referencing #37 

I added a small note in the Readme explaining the situation briefly. Maybe this can save others the hours I put in to get this working correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,40 +53,31 @@ A file "/folder/file.handlebars".
...
+  ```js
+  handlebarsLoader: {
+      partialResolver: function(partial, callback){
+          // should pass the partial's path on disk
+          // to the callback. Callback accepts (err, locationOnDisk)
+      }
+  }
+  ```
+- _config.helperResolver_ You can specify a function to use for resolving helpers. To do so, add to your webpack config:
+  `js handlebarsLoader: { helperResolver: function(helper, callback){ // should pass the helper's path on disk // to the callback if one was found for the given parameter. // Callback accepts (err, locationOnDisk) // Otherwise just call the callback without any arguments } }`
```
Comment (pcardune) [inline README.md:79]: This looks a little funny. I think you either need to put this code block back to being multiline with three back-ticks, or you need to ditch the `js` prefix. I'd prefer the multiline option myself.

Comment (juliankrieger) [reply README.md:79]: Alright, I‘ll get to it today :)

Comment (juliankrieger) [reply README.md]: @pcardune done, I don't know why, but some kind of formatter must have changed all * to _ in the markdown file. I fixed it now.

**Comment by juliankrieger** [comment]
@pcardune should be done.

**Comment by juliankrieger** [comment]
@pcardune whats holding this from being finished?

**Pull request merged**